_episodes/15.md

```diff
@@ -0,0 +1,356 @@
+
+/**
+ * Travel back to previous snapshots
+ */
+
+SELECT * FROM iceberg.logging.logs;
+
+SELECT * FROM iceberg.logging."logs$snapshots";
+
+SELECT * FROM iceberg.logging."logs@<insert-earlier-snapshot>";
+
+CALL system.rollback_to_snapshot('logging', 'logs', <insert-earlier-snapshot>)
+
+/**
+ * Back to the future snapshot
+ */
+
+SELECT * FROM iceberg.logging."logs$snapshots";
+
+SELECT * FROM iceberg.logging."logs@<insert-latest-snapshot>";
+
+CALL system.rollback_to_snapshot('logging', 'logs', <insert-latest-snapshot>)
+
+SELECT * FROM iceberg.logging.logs;
+
+SELECT * FROM iceberg.logging."logs$partitions";
+```
+
+## Question of the week: What do I do to restart the test pipeline if it fails on me?
+
+There is unfortunately no way to enable users to rerun tests without write 
+permissions to the Trino repository, so you have to do a dummy commit.
+
+This can easily be done using this one line hack 
+`git commit --amend --no-edit && git push -f`.
+
+The good news is, we have gone through some extensive lengths to identify flaky
+tests in the last year. This will identify common tests that fail across all of
+the projects that require further inspection from contributors.
+
+## Events, news, and various links
+
+Interested in supporting the Trino project, but don't know where to start? A 
+good place to start with a little less barrier to entry, is adding to the 
+documentation. We will be supporting the 
+[writing day](https://trino.io/blog/2021/04/14/wtd-writing-day.html) at the
+Write The Docs (WTD) conference this year! Join us to learn how to get involved!
+
+Come join us for the inaugural Virtual Trino meetup on April 21st in the virtual
+meetup group in your region! See [the community page](./community.html) for more
+details.
+
+ - [Americas meetup](https://www.meetup.com/trino-americas/events/277246268/)
+ - [EMEA meetup](https://www.meetup.com/trino-emea/events/277246173/)
+ - [APAC meetup](https://www.meetup.com/trino-apac/events/277246078/)
+
+At this meetup, the four Trino/Presto founders will be updating everyone on the
+state of Trino. We'll discuss the rebrand, talk about the recent features, and 
+discuss the trajectory of the project. Then we will host a hangout and AMA. Hope
+to see you all there!
+
+Blogs
+ - <https://medium.com/expedia-group-tech/a-short-introduction-to-apache-iceberg-d34f628b6799>
+ - <https://engineering.linkedin.com/blog/2021/fastingest-low-latency-gobblin>
+ - <https://medium.com/adobetech/iceberg-at-adobe-88cf1950e866>
+ - <https://medium.com/adobetech/high-throughput-ingestion-with-iceberg-ccf7877a413f>
+ - <https://medium.com/adobetech/taking-query-optimizations-to-the-next-level-with-iceberg-6c968b83cd6f>
+ - <https://thenewstack.io/apache-iceberg-a-different-table-design-for-big-data/>
+
+Videos
+ - <https://www.youtube.com/watch?v=mf8Hb0coI6o>
+ - <https://www.youtube.com/watch?v=Kxbzr7UP1dI&t=1274s>
+ - <https://www.youtube.com/watch?v=QNmSXMQ-gY4>
+
+Trino Meetup Groups
+ - Virtual
```

Comment:
I would use Trino for all of them and use actual names as 

Virtual Trino Americas
Virtual Trino EMEA

...